beacon-chain/sync/backfill/verify_test.go

```diff
@@ -30,8 +30,8 @@ func TestDomainCache(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, len(fsched), len(dc.forkDomains))
 	for i := range fsched {
-		e := fsched[i].Epoch
-		ad, err := dc.forEpoch(e)
+		n := fsched[i].Name
+		ad, err := dc.forName(n)
```

Comment:
Why this change?

Reply:
Using epoch to query fork domain is based an assumption that no two updates have a same epoch. While it's a solid one, it doesn't hold when more than two updates are underway like current case — Electra and ePBS. I think assuming there will be no duplicate name is slightly better than epoch.